services/alertmanager/registry_test.go

```diff
@@ -20,13 +20,15 @@ import (
 	"testing"
 	"time"
 
+	"github.com/go-openapi/strfmt"
 	"github.com/percona/pmm/api/alertmanager/ammodels"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 func TestRegistry(t *testing.T) {
 	nowValue, origNow := now(), now
+	nowValueCopy := nowValue
 	now = func() time.Time {
 		return nowValue
 	}
```

Comment:
I'm not sure, but maybe It's better to pass something that implements `Now() time.Time` as a parameter to `NewRegistry()`

That code is not obvious and takes me some time to figure out what is happening here.